src/mui/list/List.js

```diff
@@ -102,10 +106,9 @@ export class List extends Component {
         return this.props.location.pathname;
     }
 
-    refresh = (event) => {
+    refresh = event => {
```

Comment:
It might broke backward compatibility if someone using refresh prop in some custom action. Do you think is it acceptable?